cli/smartcontract/testdata/rpcbindings/nft-d/rpcbindings.out

```diff
@@ -0,0 +1,210 @@
+// Code generated by neo-go contract generate-rpcwrapper --manifest <file.json> --out <file.go> [--hash <hash>] [--config <config>]; DO NOT EDIT.
+
+// Package nft contains RPC wrappers for NeoFS Object NFT contract.
+package nft
+
+import (
+	"errors"
+	"fmt"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/nep11"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/nep24"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	"math/big"
+)
+
+// Hash contains contract hash.
+var Hash = util.Uint160{0x33, 0x22, 0x11, 0x0, 0xff, 0xee, 0xdd, 0xcc, 0xbb, 0xaa, 0x99, 0x88, 0x77, 0x66, 0x55, 0x44, 0x33, 0x22, 0x11, 0x0}
+
+// NftRoyaltyRecipient is a contract-specific nft.RoyaltyRecipient type used by its methods.
+type NftRoyaltyRecipient struct {
+	Address util.Uint160
+	Amount  *big.Int
+}
```

Comment:
We're doing this already for NEP-11/NEP-17, so we should also do this for NEP-24. Current NEP-14 definition is sufficient, NEP-25 will just make it more precise.

Reply:
I mean detecting the standard at the binding generator level and reusing some standard package (we don't for native packages like #3071). We never had any case of `"returntype": "Array"`, but if it's a NEP-24 contract then we can try overriding the type. At least if it's not very difficult to do so.

Reply:
> I mean detecting the standard at the binding generator level and reusing some standard package

It's already implemented. The question is only about this "extra" `NftRoyaltyRecipient` structure.

> but if it's a NEP-24 contract then we can try overriding the type

That's the problem, we have to dig into binding generator config for that. OK, I got your point, will be implemented as a part of the current PR.

Reply:
Currently implemented this way: if "extra" `NftRoyaltyRecipient` structure is used only by `royaltyInfo` (standard method) then NftRoyaltyRecipient will be dropped  (`example/nft-d`), otherwise (`example/nft-nd`) it will be saved because NftRoyaltyRecipient is used by other not standard methods.

Reply:
Yes, I believe it's the only way we have.